### PR TITLE
Fixes #5485: Save button should be disabled while actions is not finished

### PR DESCRIPTION
**Pull request by RaphaelGauthier**
https://www.rudder-project.org/redmine/issues/5485

### CODE REVIEW

**Comment by VinceMacBuche** [comment]
Can yiu rebase that PR??

**Comment by RaphaelGauthier** [comment]
PR updated

File: builder/js/ncf.js [hunk 1]
```diff
@@ -820,6 +822,7 @@ $scope.groupMethodsByCategory = function () {
     // Update selected technique if it's still the same technique
     // update technique from the tree
     var saveSuccess = function(data, status, headers, config) {
+      $scope.saving = false;
```
Comment (VinceMacBuche) [inline builder/js/ncf.js]: error should also put saving to false !!

**Comment by RaphaelGauthier** [comment]
PR updated

**Pull request merged**